projects/packages/search/changelog/add-search-chart-js

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Search: add chart.js package to dependencies
```

Comment:
Nit: We probably don't need to bump the minor version for adding an unused dependency; perhaps a patch version bump would suffice?